website/static/js/fangorn.js

```diff
@@ -1668,7 +1693,11 @@ var FGToolbar = {
             }, '')
         );
 
-        templates[toolbarModes.DEFAULT] =  m('.col-xs-12', m('.pull-right', [finalRowButtons,  m('span', generalButtons)]));
+        if (item && item.connected !== false){
```

Comment:
I wanted it to run when ```item.connected = undefined``` as well, to avoid having to initialize the connected = true. My logic specifically wants it to not equal to set value of false.